src/fermo/config.py

```diff
@@ -0,0 +1,8 @@
+""" Flask configurations """
+
+DEBUG = True
+FLASK_ENV = 'development'
+UPLOAD_FOLDER = 'src/fermo/uploads/'  # temporary until database is implemented
+ALLOWED_EXTENSION = {'json', 'mgf', 'csv'}
+# todo: secret key should be individual for every session
```

Comment:
Better to put 'to-do's in a separate file (e.g. a TO_DO.md file) or on GitHub with a prioritiy system to prevent cluttering of code with comments